src/components/Catalog/Catalog.jsx

```diff
@@ -0,0 +1,78 @@
+
+import '../../style/Catalog.css';
+import { Component } from 'react';
+import {Link } from 'react-router-dom'
+import Movie from './Movie'
+class Catalog extends Component {
+    constructor()
+    {
+        super()
+        this.state = {
+        search: ""
+        }
+    }
+
+    handleSearchBar = (event) =>
+    {
+        const {value, name} = event.target
+        this.setState(
+            {
+                [name]: value 
+            }
+        )
+        
+    }
+  render()
+  {
+
+      let name = this.props.match.params.name
```

Comment:
`const {name} = this.props.match.params`
`const {movies, users} = this.props`
`const {search} = this.state.search`
then you don't need to keep repeating this and props